docs/reference/inference/service-cohere.asciidoc

```diff
@@ -131,6 +131,7 @@ Specify whether to return doc text within the results.
 `top_n`::
 (Optional, integer)
 The number of most relevant documents to return, defaults to the number of the documents.
+If `top_n` set to a greater number than the number of document to rerank, the request will fail.
```

Comment:
Thanks Istvan! This is tricky, so I think we need to be a bit more specific with the wording. I think it could be:

```suggestion
If using the `text_similarity_reranker` retriever in a query with a cohere inference endpoint, `top_n` must be greater than or equal to the `rank_window_size`.
```

What do you think @demjened? Do you have any ideas about how to word this?

Reply:
@maxhniebergall Something along these lines? I.e. mention that this is only a problem if `top_n` configured:
```suggestion
If this inference endpoint is used in a `text_similarity_reranker` retriever query and `top_n` is set, it must be greater than or equal to `rank_window_size` in the query.
```

Reply:
That LGTM